integration-tests/emulated/tests/bridges/bridge-hub-kusama/src/tests/asset_transfers.rs

```diff
@@ -206,15 +165,8 @@ fn send_ksms_from_asset_hub_kusama_to_asset_hub_polkadot() {
 	let ksms_in_reserve_on_ahk_after =
 		<AssetHubKusama as Chain>::account_data_of(sov_ahp_on_ahk.clone()).free;
 
-	dbg!(&sender_ksms_after);
-	dbg!(&sender_ksms_before);
-	dbg!(&amount);
-	dbg!(&delivery_fees);
-	dbg!(&receiver_ksms_after);
-	dbg!(&receiver_ksms_before);
-
 	// Sender's balance is reduced
-	assert_eq!(sender_ksms_after, sender_ksms_before - amount - delivery_fees);
```

Comment:
Is this no longer the case or are you just making the test less brittle?

Reply:
It was not correct, at least, it missed the subtraction of `execution_fees`. I tried to add them as well, but [`let result = Runtime::dry_run_call(origin, call).unwrap();`](https://github.com/polkadot-fellows/runtimes/pull/397/files#diff-44fd5c77737282dbb2dfbba20d499bbc921d1d00943b304fbacc885c82c7edcdR60) did not return `actual_weight`, so it is not possible to calculate `WeightToFee`. Additionally, there was some calculation of `intermediate_delivery_fees_amount` from BHK to BHP, which is not relevant. So yes, keep it simple. At least I added asserts to check that the dry-run succeeded and the message is ready to be relayed over bridge to the BHP.

Reply:
This was added by @franciscoaguirre, fee estimation doesn't yet work with BridgeHub exporters: https://github.com/polkadot-fellows/runtimes/pull/380#issuecomment-2233024669

The check will be added back when that is fixed as well.